Resources/public/scss/modules/sub-items-list/_no.items.scss

```diff
@@ -0,0 +1,5 @@
+.c-no-items {
+    padding: 0 calculateRem(32px);
+    font-style: italic;
+    color: $ez-color-base-light;
```

Comment:
For the rest of empty tables we are applying:
`font-size: calculateRem(14px);`
`color: $ez-color-base-dark;`
It would be good to have them here too for consistency purposes.